### ISSUE TITLE
Split out cargo-frc into separate repo?

### ISSUE THREAD

**Issue by auscompgeek** [first-rust-competition/first-rust-competition#79]
cargo-frc probably won't change much, or as much as the wpilib crate.

**Comment by Lytigas** [comment]
I agree they wont change as much. I tend to prefer a mono-repo, though, where everything deploys together and has been tested together. A deploy tool obviously matters less for that, but some things, like the version of `wpilib` used in a `cargo frc init` project, would change together. In general, I'd be in favor of moving more things into a single repo, rather than splitting things apart.

**Comment by auscompgeek** [comment]
Oh, there's a `cargo frc init`? Hmm. I guess in that case unless we can come up with a good way to make that work without manually bumping versions I guess keeping it here makes sense.

**Comment by Lytigas** [comment]
It's planned... Doesn't exist at the moment.
